src/call-api.test.ts

```diff
@@ -0,0 +1,394 @@
+import * as adapter from "@api3/airnode-adapter";
+import * as node from "@api3/airnode-node";
+import * as ois from "@api3/airnode-ois";
+import { ethers } from "ethers";
+import { readApiValue } from "./call-api";
+
+describe("readApiValue", () => {
+  const airnodeAddress = "0xA30CA71Ba54E83127214D3271aEA8F5D6bD4Dace";
+  const oises: ois.OIS[] = [
+    {
+      oisFormat: "1.0.0",
+      version: "1.2.3",
+      title: "Currency Converter API",
+      apiSpecifications: {
+        servers: [
+          {
+            url: "http://localhost:5000",
+          },
+        ],
+        paths: {
+          "/convert": {
+            get: {
+              parameters: [
+                {
+                  in: "query",
+                  name: "from",
+                },
+                {
+                  in: "query",
+                  name: "to",
+                },
+                {
+                  in: "query",
+                  name: "amount",
+                },
+                {
+                  in: "query",
+                  name: "date",
+                },
+              ],
+            },
+          },
+        },
+        components: {
+          securitySchemes: {
+            "Currency Converter Security Scheme": {
+              in: "query",
+              type: "apiKey",
+              name: "access_key",
+            },
+          },
+        },
+        security: {
+          "Currency Converter Security Scheme": [],
+        },
+      },
+      endpoints: [
+        {
+          name: "convertToUSD",
+          operation: {
+            method: "get",
+            path: "/convert",
+          },
+          fixedOperationParameters: [
+            {
+              operationParameter: {
+                in: "query",
+                name: "to",
+              },
+              value: "USD",
+            },
+          ],
+          reservedParameters: [
+            {
+              name: "_type",
+              fixed: "int256",
+            },
+            {
+              name: "_path",
+              fixed: "result",
+            },
+            {
+              name: "_times",
+              default: "1000000",
+            },
+          ],
+          parameters: [
+            {
+              name: "from",
+              default: "EUR",
+              operationParameter: {
+                in: "query",
+                name: "from",
+              },
+            },
+            {
+              name: "amount",
+              default: "1",
+              operationParameter: {
+                name: "amount",
+                in: "query",
+              },
+            },
+          ],
+          testable: true,
```

Comment:
Not sure what version is this for but this no longer belongs to the OSI section in `v0.4.x`.

Reply:
Airkeeper currently only works with v0.3.1 and we'll probably update it's dependencies after the ETH Denver hackathon.